XamarinCommunityToolkit/Behaviors/UserStoppedTypingBehavior.shared.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+    public class UserStoppedTypingBehavior : BaseBehavior
+    {
+        CancellationTokenSource tokenSource;
+
+        #region Bindable Properties
+
+        public static readonly BindableProperty CommandProperty = BindableProperty.Create(nameof(Command), typeof(ICommand), typeof(UserStoppedTypingBehavior));
+        public ICommand Command
+        {
+            get => (ICommand)GetValue(CommandProperty);
+            set => SetValue(CommandProperty, value);
+        }
+
+        public static readonly BindableProperty StoppedTypingThresholdProperty = BindableProperty.Create(nameof(StoppedTypingThreshold), typeof(int), typeof(UserStoppedTypingBehavior), defaultValue: 1000);
+        public int StoppedTypingThreshold
+        {
+            get => (int)GetValue(StoppedTypingThresholdProperty);
+            set => SetValue(StoppedTypingThresholdProperty, value);
+        }
+
+        public static readonly BindableProperty AutoDismissKeyboardProperty = BindableProperty.Create(nameof(AutoDismissKeyboard), typeof(bool), typeof(UserStoppedTypingBehavior), defaultValue: false);
+        public bool AutoDismissKeyboard
```

Comment:
@jfversluis @sthewissen @Char0394 
Perhaps it's only my personal preference, but I prefer to use "Is", "Are", "Should" as infix for all boolean properties.

e.g. ShouldDismissKeyboardAutomatically or IsKeyboardAutoDismissing